doc/api/repl.markdown

```diff
@@ -21,21 +21,21 @@ dropped into the REPL. It has simplistic emacs line-editing.
     2
     3
 
-For advanced line-editors, start io.js with the environmental variable
+For advanced line-editors, start Node.js with the environmental variable
 `NODE_NO_READLINE=1`. This will start the main and debugger REPL in canonical
 terminal settings which will allow you to use with `rlwrap`.
 
 For example, you could add this to your bashrc file:
 
-    alias iojs="env NODE_NO_READLINE=1 rlwrap iojs"
+    alias node="env NODE_NO_READLINE=1 rlwrap node"
 
 ## Persistent History
 
-By default, the REPL will persist history between `iojs` REPL sessions by saving
+By default, the REPL will persist history between `node` REPL sessions by saving
 to a `.node_repl_history` file in the user's home directory. This can be
 disabled by setting the environment variable `NODE_REPL_HISTORY=""`.
 
-Previously in io.js v2.x, REPL history was controlled by using a
+Previously in Node.js/io.js v2.x, REPL history was controlled by using a
```

Comment:
We can expect a long tail of v2.x users since v3 has breakage and performance issues. v4 will be the first official LTS, but just saying.